client/src/hooks/api.ts

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
 import Fuse from 'fuse.js';
-import api from 'store/api';
+import api from 'store/admin/api';
```

Comment:
I decided that it was easier / more secure to create two separate redux stores, there's now a register store and an admin store.